tests/program.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+include $_SERVER['argv'][1];
```

Comment:
isn't it just `$argv[1]` ?

Reply:
It seemed not to work before, but now it does. Maybe it was due to some other reason that I missed.. Anyway, I changed it.